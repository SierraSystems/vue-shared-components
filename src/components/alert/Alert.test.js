import Alert from "./Alert.vue";
import CheckboxMarked from "mdi-vue/CheckboxMarked.vue";
import { mount } from "@vue/test-utils";

describe("Alert component", () => {
  test("matches the success snapshot", () => {
    const wrapper = mount(Alert, {
      propsData: {
        styling: "bcgov-success-background",
        element: "This is a success message!",
        alertType: "success",
        icon: CheckboxMarked,
      }
    });

    expect(wrapper.html()).toMatchSnapshot();
  });
});
