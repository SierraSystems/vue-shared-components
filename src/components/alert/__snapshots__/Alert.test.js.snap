// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Alert component matches the success snapshot 1`] = `
<div class="bcgov-display-box bcgov-success-background">
  <div class="bcgov-display-icon">
    <div>
      <div class="bcgov-success-color"><span role="img" aria-label="CheckboxMarked icon" class="mdi mdi-checkbox-marked"><svg fill="currentColor" width="24" height="24" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><!----> <path d="M10,17L5,12L6.41,10.58L10,14.17L17.59,6.58L19,8M19,3H5C3.89,3 3,3.89 3,5V19A2,2 0 0,0 5,21H19A2,2 0 0,0 21,19V5C21,3.89 20.1,3 19,3Z"></path></svg></span></div>
      <!---->
      <!---->
      <!---->
    </div>
  </div>
  <div class="bcgov-display-right-element">
    This is a success message!
  </div>
</div>
`;
